lib/main.dart

```diff
@@ -34,7 +37,17 @@ class MyHomePage extends StatelessWidget {
             }),
       ),
       body: Center(
-        child: Text(FlutterConfig.get('SECRET')),
+        child: Column(
+          children: [
+            SizedBox(height: 24,),
+            FractionallySizedBox(
+              widthFactor: 0.5,
+              child: Image.asset(Assets.images.nimbleLogo.path, fit: BoxFit.fitWidth),
```

Comment:
it seems like you forgot to commit this image file 🙋‍♂️

Reply:
@doannimble just in case you missed this comment ⭕

Reply:
@sleepylee I've added.